docs/quick-start/tips-and-tricks.md

```diff
@@ -0,0 +1,72 @@
+-  Clear over a time: ``<n>`` then \<Clear\> will clear over that time
+-  \<Wheel @\> Keys: Allow fast Access to Freeze, Off, Or on for the attribute active for that wheel. Tap it twice for 100% of a percentage based attribute (such as Intensity, RGB or CMY). [(more)](../controlling-fixtures/changing-fixture-attributes.md#adjusting-attributes-with-the--buttons)
+-  How to see/select cells on multi cell fixture: Use \<Unfold\> then tap fixture.
+-  Group Pattern Selection: Click and hold \<Group\> then select from the soft keys Odd/Even etc
+-  Quick Pattern Selection: Select fixtures or Group then click \<ALL\> to select within selected fixtures a pattern such as
+odd, even, random – etc
+-  Quickly set record type: press \<Record\> once for Cue, twice for chase, three times for cue list
+-  \<AVO\> +\<Palette\> = Quick access to the record palette menu
+-  \<AVO\> +\<Group\> = Quick access to the record group menu
+-  \<AVO\> +\<Macro\> = Quick access to the record macro menu
+-  Record time into palettes/cues: Use the \<Time\> key to put times in the programmer
+-  \<AVO\> + \<COPY\> = Move
+-  Want to Move/Copy/ Delete lots of items: Use the \<Latch Menu\> after \<COPY\>, \<MOVE\>, or \<Delete\> to stay in that
+menu
+
+## Playback and running a show
+
+-  Set BPM/Rate directly: ``<n>`` then tap one of the master's keys (any of them) and it will set that as the BPM/Rate %
+-  Fire Macro ``<n>`` times: Type number then fire macro, macro will be played that many times
+-  Change fade time of next Cue: ``<n>`` \<GO\> - fades in the next cue in a cuelist in n seconds, overriding the programmed
+fade time
+-  Release all running playbacks: Double tap \<Release\> to release ALL running playbacks
+-  Legend Wing Pages: Set Legend then \<Go Page\> on the wing.
+
+## Views, menus and windows
+
+-  Quick open Patch View: \<Open/View\> + \<Patch\>
+-  Quick Open Fixtures Window: \<Open/View\> + \<Fixture\>
+-  Quick Open Groups Window: \<Open/View\> + \<Group\>
+-  Quick Open Macro’s Window: \<Open/View\> + \<Macro\>
+-  Quick Open Attribute Editor: \<Open/View\> + \<Options\>
+-  Quick Open Active Playbacks: \<Open/View\> + \<Off\>
+-  Quick Open Gobo and Beam Palettes Window: \<Open/View\> + \<Beam/Open/View\> + \<Gobo\>
+-  Quick Open Colour Palettes Window: \<Open/View\> + \<Colour\>
+-  Quick Open Position Palettes Window: \<Open/View\> + \<Postion\>
+-  Quick Open Shape Palettes Window: \<Open/View\> + \<Shapes\>
+-  Quick Open Intensity View: \<Open/View\> + \<Intenisity\>
+-  Quick Open Cue List View of connected Cue List: \<Open/View\> + \<Connect/Cue\>
+-  Quick exit menus to the top: \<AVO\> + \<EXIT\>, This will also exit the Unfold Menu
+-  Search deep in menus: use the search box to find menu options faster.
+-  Options missing in a Menu: Check to see if you have typed in the search box in the prompt area by accident
+-  Window too small to Adjust: \<Open/View\> + \<Min/Max\> to resize window
```

Comment:
As of 15, this should now be fixed so that the resize button is always visible. Maybe useful for users of older versions though